### PR TITLE
Create export to excel functionality.

### PR DESCRIPTION
**Pull request by MTsatsarov**
Closes https://github.com/SoftUni-Internal/exam-systems-issues/issues/1212

**Summary of the changes made**:
1. Added excel service with required functionality.
2. New method added in the grid service.
3.  Fe button and requests to take excel files created.

### CODE REVIEW

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/contestsAdminService.ts [hunk 1]
```diff
@@ -71,6 +72,19 @@ export const contestService = createApi({
                 body: contestAdministrationModel,
             }),
         }),
+
+        getContestsExcelFile: builder.query<IFileModel, IGetAllAdminParams>({
```
Comment (gogo4ds) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/contestsAdminService.ts:76]: Isn't this name inconsistent with the others?

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/participantsAdminService.ts [hunk 2]
```diff
@@ -41,6 +41,18 @@ export const participantsAdminService = createApi({
...
+            query: ({ filter, page, itemsPerPage, sorting }) => ({
+                url: `/${EXCEL_RESULTS_ENDPOINT}`,
+                params: {
+                    filter,
+                    page,
+                    itemsPerPage,
+                    sorting,
+                },
+            }),
+            keepUnusedDataFor: 5,
```
Comment (gogo4ds) [inline Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/participantsAdminService.ts:54]: Why here is 5 and on other places 0 or 10?

**Review by gogo4ds — CHANGES_REQUESTED** [review]

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/contestsAdminService.ts (hunk 1 above)
Comment (MTsatsarov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/contestsAdminService.ts:76]: Fixed

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/participantsAdminService.ts (hunk 2 above)
Comment (MTsatsarov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/redux/services/admin/participantsAdminService.ts:54]: Fixed

File: Servers/UI/OJS.Servers.Ui/ClientApp/src/common/urls/administration-urls.ts [hunk 3]
```diff
@@ -8,6 +8,8 @@ export const DELETE_ENDPOINT = 'Delete';
 export const GET_ENDPOINT = 'Get';
 export const GETALL_ENDPOINT = 'GetAll';
 
+export const EXCEL_RESULTS_ENDPOINT = 'GetExcelResults';
```
[the remark this answers is not in the document]
Comment (MTsatsarov) [reply Servers/UI/OJS.Servers.Ui/ClientApp/src/common/urls/administration-urls.ts]: Fixed.

**Review by gogo4ds — APPROVED** [review]

**Review by gogo4ds — APPROVED** [review]

**Pull request merged**